recipes/bamnostic/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "bamnostic" %}
+{% set version = "0.4.2b9" %}
+{% set sha256 = "a205563e4c9fed1deca50dfb096b59387bc98ec656c5b97327a3d3c119ec5db5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```

Comment:
I believe the `build` section only needs `python` and `pip`. The others aren't necessary to actually build the package.

Reply:
Will remove!